syne_tune/backend/trial_status.py

```diff
@@ -81,8 +81,8 @@ def cost(self):
         else:
             return None
 
-    def metrics_df(self) -> pd.DataFrame:
-        if len(self.metrics) == 0:
-            return pd.DataFrame()
-        else:
-            return pd.DataFrame(data=self.metrics)
+    # def metrics_df(self) -> pd.DataFrame:
+    #     if len(self.metrics) == 0:
+    #         return pd.DataFrame()
+    #     else:
+    #         return pd.DataFrame(data=self.metrics)
```

Comment:
Good catch, it does not seem used. We can remove the lines altogether though (preferable than adding dead comments https://gist.github.com/wojteklu/73c6914cc446146b8b533c0988cf8d29).